requirements.txt

```diff
@@ -1 +1,2 @@
 ops ~= 2.5
```

Comment:
[Suggestion] I'd suggest adding `dev-requirements.txt` file which contains version of `tox`.